lib/modules/datasource/github-tags/cache.spec.ts

```diff
@@ -28,6 +28,8 @@ describe('modules/datasource/github-tags/cache', () => {
             type: 'Tag',
             target: {
               hash: 'abc',
+            },
+            tagger: {
               releaseTimestamp: '2020-04-09T10:00:00.000Z',
```

Comment:
Not sure if the date parser can do that automatically, but maybe we should adjust the test as well to reflect the data responded from graphql e.g. `2020-04-09T06:00:00.000-04:00` to ensure that it is correctly parsed

Reply:
I assumed downstream code has no problems with different timezone formats

Reply:
Wasn't sure about that, but checked as well and it seems that ` DateTime.fromISO(date);` does correctly manages to parse the date, so everything good 👍